### ISSUE TITLE
add model zoo

### ISSUE THREAD

**Issue by ChenMinQi** [facebookresearch/ClassyVision#323]
## 🚀 Feature

model zoo just like detectron2

## Motivation

We can quickly know the accuracy of each model.
## Pitch

just like this, [detectron2 model zoo](https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md)

**Comment by dsp6414** [comment]
i use torchvision.models for model zoo
[code]
def Create_Model_From_Zoo(num_classes=4):
    model = models.resnet50(pretrained=True)
    num_ftrs = model.fc.in_features
    model.fc = nn.Linear(num_ftrs,num_classes)
    classy_model = ClassyModel.from_model(model)
    return classy_model

...
config_file='./configs/test.json'
config = load_json(config_file)

model = Create_Model_From_Zoo(num_classes=4)
task = (
    My_ClassificationTask()
    .set_loss(build_loss(config["loss"]))
    .set_model(model)
    .set_optimizer(build_optimizer(config["optimizer"]))
    .set_meters([AccuracyMeter(topk=[1])])
)

...
